CHANGES_UNRELEASED.md

```diff
@@ -10,14 +10,13 @@
 
 - Login records with a `httpRealm` attribute will now have their `usernameField` and `passwordField
   properties silently cleared, to help ensure data consistency. ([#2158](https://github.com/mozilla/application-services/pull/2158))
-- The Android bindings now collect some basic performance and quality metrics via Glean.
-  Applications that submit telemetry via Glean *must* request a data review for these new metrcs
-  before upgrading to the latest version of the logins component. See the component README.md
-  for more details. ([#2225](https://github.com/mozilla/application-services/pull/2225))
 
 ### What's new
 
 - Added invalid character checks from Desktop to `LoginsStorage.ensureValid` and introduced `INVALID_LOGIN_ILLEGAL_FIELD_VALUE` error. ([#2262](https://github.com/mozilla/application-services/pull/2262))
+- The Android bindings can now collect some basic performance and quality metrics via Glean.
+  Collection is off by default, but consuming apps that submit telemetry via glean are encouraged
```

Comment:
Is this required by some specific product you're currently targetting?

Reply:
It's driven by the push-back on integration into a-c as a breaking change here: https://github.com/mozilla-mobile/android-components/pull/5223